### PR TITLE
refactor: change expression check to option chain

### LINKED ISSUE #784: Prefer using an optional chain expression instead, as it's more concise and easier to read.
## Why this issue?
https://sonarcloud.io/project/issues?resolved=false&id=asyncapi_asyncapi-react&open=AYrVnLzdi9mf_H5ePyyP&tab=why

## Where's the issue? 
https://sonarcloud.io/project/issues?resolved=false&id=asyncapi_asyncapi-react&open=AYrVnLzdi9mf_H5ePyyP&tab=code

### PR DESCRIPTION
**Pull request by syedzubeen**
**Description**

Changed the expression to an optional chain

**Related issue(s)**
Fixes #784

### CODE REVIEW

**Comment by derberg** [comment]
@syedzubeen Thanks for the PR. When you contribute to any project, especially open source, you should open a PR only once you are familiarized with contributing guide like https://github.com/asyncapi/asyncapi-react/blob/master/CONTRIBUTING.md

- please adjust your PR title:
   - use [imperative mood](https://cbea.ms/git-commit/#imperative) in PR title
   - use proper conventional commits prefix (you have a bot comment with details - in your case it is `refactor`

**Comment by syedzubeen** [comment]
@derberg my bad. Should've gone through it first.
Would just like to point out that [CONTRIBUTING.md](https://github.com/asyncapi/asyncapi-react/blob/master/CONTRIBUTING.md) has two dead links for:

- [Naming and architecture convention](https://github.com/asyncapi/asyncapi-react/blob/master/DEVELOPMENT.md#naming--architecture-convention)
- [Project structure](https://github.com/asyncapi/asyncapi-react/blob/master/development-guide.md/#project-structure)

File: library/src/helpers/message.ts [hunk 1]
```diff
@@ -84,8 +84,9 @@ export class MessageHelpers {
     }
 
     const headers = msg.headers();
-    if (headers && headers.examples()) {
-      return headers.examples()?.map(example => ({ example }));
+    const examples = headers?.examples?.();
+    if (examples) {
+    return examples.map(example => ({ example }));
```
Comment (derberg) [inline library/src/helpers/message.ts:89]: please add proper indentation

**Review by derberg — COMMENTED** [review]
left one small comment

please, if you can, also open a separate PR to fix the contrib guide with proper links
- https://github.com/asyncapi/asyncapi-react/blob/master/docs/development/guide.md#naming-and-architecture-convention
- https://github.com/asyncapi/asyncapi-react/blob/master/docs/development/guide.md#project-structure

thanks for pointing it out

File: library/src/containers/AsyncApi/Standalone.tsx [hunk 2]
```diff
@@ -32,7 +32,6 @@ class AsyncApiComponent extends Component<AsyncApiProps, AsyncAPIState> {
     if (parsedSpec) {
       this.state = { asyncapi: parsedSpec };
     }
-    return;
```
Comment (derberg) [inline library/src/containers/AsyncApi/Standalone.tsx:35]: please revert this change,
https://github.com/asyncapi/asyncapi-react/pull/802 PR handles that change

File: library/src/helpers/message.ts (hunk 1 above)
Comment (syedzubeen) [reply library/src/helpers/message.ts]: done

File: library/src/containers/AsyncApi/Standalone.tsx (hunk 2 above)
Comment (syedzubeen) [reply library/src/containers/AsyncApi/Standalone.tsx]: done
